src/feed-list.scss

```diff
@@ -0,0 +1,127 @@
+@import "./mixins";
+
+$block: #{$fd-namespace}-feed-list;
+
+.#{$block} {
+  $fd-feed-list-padding-start: 1rem;
+  $fd-feed-list-padding-end: 0.5rem;
+  $fd-feed-list-margin-start: 0.5rem;
+  $fd-feed-list-margin-end: 0;
+  $fd-feed-list-icon-color: var(--sapContent_IconColor);
+
+  @include fd-reset();
+  @include fd-flex();
+
+  background-color: var(--sapField_Background);
+  max-width: 40rem;
+
+  .#{$block}__thumb {
+    @include fd-reset();
+
+    display: inline-flex;
+    margin-left: $fd-feed-list-margin-end;
+    margin-right: $fd-feed-list-margin-start;
+
+    @include fd-rtl() {
+      margin-left: $fd-feed-list-margin-start;
+      margin-right: $fd-feed-list-margin-end;
+    }
+  }
+
+  &__body {
+    @include fd-reset();
+
+    display: flex;
+    padding: 1rem $fd-feed-list-padding-end 1rem $fd-feed-list-padding-start;

```

Comment:
Shouldn't you add padding 1rem for whole container? 
![image](https://user-images.githubusercontent.com/26483208/95840251-3733a200-0d44-11eb-8715-6c722088bdfa.png)